hopsworks-web/yo/app/scripts/controllers/loginCtrl.js

```diff
@@ -59,14 +59,6 @@ angular.module('hopsWorksApp')
             self.openIdProviders = [];
 
 
```

Comment:
We are not showing the default password in tool tip anymore?
Then you should remove https://github.com/tkakantousis/hopsworks/blob/HOPSWORKS-672/hopsworks-web/yo/app/views/login.html#L67